chain/rosetta-rpc/src/utils.rs

```diff
@@ -94,6 +94,15 @@ where
     }
 }
 
+impl<T> PartialEq for BlobInHexString<T>
+where
+    T: AsRef<[u8]> + From<Vec<u8>>,
+{
+    fn eq(&self, other: &Self) -> bool {
+        self == other
```

Comment:
Hm, isn't this an infinite recursion?

Reply:
What would be a good way to mitigate it?

Reply:
`self.0.as_ref() == other.0.as_ref()`, and it'd be great if we have a test that acutally shows an SO here